CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 6.2.0 (Apr 14, 2022)
+
+FEATURES:
+
+* Added new `artifactory_managed_user` resource which is an alias of existing `artifactory_user`.
+* Added new `artifactory_unmanaged_user` resource with `password` attribute being required and no automatic password generation.
```

Comment:
No, these are the opposite. Managed requires a password and unmanaged doesn't

Reply:
Ok will swap over.